modules/aws/ses/main.tf

```
@@ -0,0 +1,4 @@
+resource "aws_ses_email_identity" "example" {
```

`example ` って名前だけが気になった！ 

送信元のメールアドレスだから`from_email` とかはどうだろう🐱？？

```suggestion
resource "aws_ses_email_identity" "from_email" {
```

---

確認ありがとう！サンプルコードのままになっていたから、修正したよ！